build.sbt

```diff
@@ -28,7 +28,6 @@ addCommandAlias(
 ThisBuild / description                         := "Explore"
 Global / onChangedBuildSource                   := ReloadOnSourceChanges
 ThisBuild / scalafixDependencies += "edu.gemini" % "lucuma-schemas_3" % Versions.lucumaSchemas
-ThisBuild / scalafixScalaBinaryVersion          := "2.13"
```

Comment:
Why was this removed?

Reply:
It was causing a deprecated warning with the new scalafix.